starter-code/scripts/views/adminPage.js

```diff
@@ -2,10 +2,12 @@
 
 articleView.renderAdminPage = function() {
   var statsRender = Handlebars.compile($('#stats-template').html());
-  Article.numWordsByAuthor().forEach(function(numWordsObj) {
-    $('.author-stats').append(statsRender(numWordsObj));
-  });
-
+  function updateAuthorStats(){
+    Article.numWordsByAuthor().forEach(function(numWordsObj) {
+      $('.author-stats').append(statsRender(numWordsObj));
+    });
+  }
+  updateAuthorStats();
```

Comment:
You shouldn't need to wrap this in a function and then call it. Is there a reason that you did this? articleView.renderAdminPage is being called inside of Article.fetchAll (it is passed into Article.fetchAll at the bottom of adminPage.js), and that will compile our Handlebars and append our stats.